lib/ui/widgets/settingsView/settings_advanced_section.dart

```diff
@@ -22,84 +20,14 @@ class SAdvancedSection extends StatelessWidget {
     return SettingsSection(
       title: 'settingsView.advancedSectionTitle',
       children: <Widget>[
-        SManageApiUrlUI(),
-        SManageSourcesUI(),
-        // SManageKeystorePasswordUI(),
         SAutoUpdatePatches(),
         SEnablePatchesSelection(),
-        SExperimentalUniversalPatches(),
         SExperimentalPatches(),
-        ListTile(
-          contentPadding: const EdgeInsets.symmetric(horizontal: 20.0),
-          title: I18nText(
-            'settingsView.regenerateKeystoreLabel',
-            child: const Text(
-              '',
-              style: TextStyle(
-                fontSize: 20,
-                fontWeight: FontWeight.w500,
-              ),
-            ),
-          ),
-          subtitle: I18nText('settingsView.regenerateKeystoreHint'),
-          onTap: () => _showDeleteKeystoreDialog(context),
-        ),
-        ListTile(
-          contentPadding: const EdgeInsets.symmetric(horizontal: 20.0),
-          title: I18nText(
-            'settingsView.deleteTempDirLabel',
-            child: const Text(
-              '',
-              style: TextStyle(
-                fontSize: 20,
-                fontWeight: FontWeight.w500,
-              ),
-            ),
-          ),
-          subtitle: I18nText('settingsView.deleteTempDirHint'),
-          onTap: () => _settingsViewModel.deleteTempDir(),
-        ),
-        ListTile(
-          contentPadding: const EdgeInsets.symmetric(horizontal: 20.0),
-          title: I18nText(
-            'settingsView.deleteLogsLabel',
-            child: const Text(
-              '',
-              style: TextStyle(
-                fontSize: 20,
-                fontWeight: FontWeight.w500,
-              ),
-            ),
-          ),
-          subtitle: I18nText('settingsView.deleteLogsHint'),
-          onTap: () => _settingsViewModel.deleteLogs(),
-        ),
+        SExperimentalUniversalPatches(),
+        SManageSourcesUI(),
+        SManageApiUrlUI(),
+        // SManageKeystorePasswordUI(),
```

Comment:
I wasn't sure if I should move the `SManageKeystorePasswordUI()` comment to the Import & Export section so I just kept it here